CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
    ```
 
-3. Get the code
+3. Get, build and test the code
 
    ```bash
    git clone https://github.com/citusdata/citus.git
+
+   cd citus
+   ./configure
+   make
+   sudo make install
+   cd src/test/regress
+   make check
    ```
 
-4. Build and test
+#### Linux with Apt-Get (Ubuntu, Debian)
+
+1. Install build dependencies
 
    ```bash
+  sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
```

Comment:
I like the new way. It's nice to simplify the sudo part. Had to use double-quotes for the interpolation but otherwise it worked great.